File: 알고리즘/1/완주하지 못한 선수.py
def solution(participant, completion):
    dict1={}
    answer=''
    for ele in participant:
        dict1[ele] = dict1.get(ele, 0)+1
    for ele in completion:
        dict1[ele]-=1
            
    for i, ele in dict1.items():
        if ele>0:
            return i

File: 알고리즘/1/완주하지 못한 선수.cpp
#include <bits/stdc++.h>

using namespace std;

string solution(vector<string> participant, vector<string> completion) {
    string answer = "";
    unordered_map<string, int> map;

    for(auto ele : participant){
        if(map.find(ele)==map.end()){
            map.insert({ele, 1});
        } else{
            map[ele]+=1;
        }
    }

    for(auto ele: completion){
        map[ele]-=1;
    }

    for(auto ele : participant){
        if(map[ele]>0) return ele;
    }
}